docs/reference/painless-api-reference.asciidoc

```diff
@@ -0,0 +1,17 @@
+["appendix",role="exclude",id="painless-api-reference"]
```

Comment:
Ah. When I first wrote this I was going under the assumption that we didn't want it in the ToC but I'm not really sure why I assumed that...